docs/client-specifications/feature-matrix.md

```diff
@@ -0,0 +1,51 @@
+<img src="https://docs.momentohq.com/img/logo.svg" alt="logo" width="400"/>
+
+# Momento Client SDKs: Feature Matrix
+
+This page should serve as a checklist for developers who are preparing the client library
+for a given language for a 1.0 release.
+
+## Features / Requirements
+
+* `Release`: Repo has been set up according to our [SDK release process](https://github.com/momentohq/standards-and-practices/blob/main/docs/momento-sdk-release-process.md) and binary artifacts are published to the appropriate
+             package repository for the language.
+* `Get/Set APIs`: Support for basic get/set APIs, with Strings or byte arrays.
+* `Examples`: Repo includes example code that is tested in CI
+* `Error Handling`: SDK has been updated to be consistent with [Momento error handling specifications](https://github.com/momentohq/standards-and-practices/blob/main/docs/client-specifications/error-handling.md).
+* `Config`: Provides Configuration interface that can be used to adjust relevant settings as per [Momento configuration specification](https://github.com/momentohq/standards-and-practices/blob/main/docs/client-specifications/configuration.md).
+* `Retries`: Basic retry support for failed requests, and the ability for a user to configure a custom retry strategy.
+* `LoadGen`: Includes some very minimal scripts that can be used to generate load through the SDK; these are intended for
+             use in tuning for pre-built configurations, and also to serve as a starting point for users who might want to
+            modify them to simulate their own load.
+* `Pre-Built Configs`: Pre-built configurations for Laptop and In-Region, which have been tuned to ensure good performance in
+               each environment.
+* `Logging`: Supports logging configuration; logging at different verbosity levels and to different destinations, ideally
+             via the idiomatic logging framework for the language (if one exists).
+* `Debug Logging`: includes log statements for logging the execution and success or failure of every API call, for debugging
```

Comment:
S&P or Notion doc? How is "Logging" distinct from "Debug Logging"?

Reply:
I just need names that I can use in the matrix.  Do the descriptions above not make it clear what the difference is?  One is basic logging integration with the language's logging platform; the other is actual trace logs for every api call.

Reply:
Got it.